microsoft/testsuites/core/dns.py

```diff
@@ -48,13 +48,28 @@ def verify_dns_name_resolution_after_upgrade(self, node: Node) -> None:
             )
             if 0 != cmd_result.exit_code:
                 node.os.install_packages("unattended-upgrades")
+            if not isinstance(node.os, Ubuntu):
```

Comment:
Does it cover Debian only? If so, use `type(node.os) == Debian)`

Reply:
yes, updated.